plugins/azure/loadbalancer/lbHasTags.js

```diff
@@ -10,12 +10,26 @@ module.exports = {
     link: 'https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources',
     recommended_action: 'Modify affected load balancers and add tags.',
     apis: ['loadBalancers:listAll'],
+    settings: {
+        ignore_internal_loadbalancers: {
```

Comment:
```suggestion
        ignore_internal_loadbalancers: {
```
setting name should be unique across all plugins